<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="stateWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="backButton">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="icon">
         <iconset resource="ImgSrc.qrc">
          <normaloff>:/img/misc/back.png</normaloff>:/img/misc/back.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>72</width>
          <height>23</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="messageLabel"/>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="pixmap">
         <pixmap resource="ImgSrc.qrc">:/img/misc/date.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="timeLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>当前时间</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="neteorkPhoto">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="pixmap">
         <pixmap resource="ImgSrc.qrc">:/img/misc/networkState.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="netStatusLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>网络正常</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="pixmap">
         <pixmap resource="ImgSrc.qrc">:/img/misc/power.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="powerLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>电源正常</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_temperatureIcon">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="pixmap">
         <pixmap resource="ImgSrc.qrc">:/img/misc/temperature.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="temperature">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>温度：25℃</string>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>powerLabel</zorder>
     <zorder>netStatusLabel</zorder>
     <zorder>timeLabel</zorder>
     <zorder>label_4</zorder>
     <zorder>neteorkPhoto</zorder>
     <zorder>label_6</zorder>
     <zorder>label_temperatureIcon</zorder>
     <zorder>temperature</zorder>
     <zorder>messageLabel</zorder>
     <zorder>backButton</zorder>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="mainWidget" native="true">
     <layout class="QVBoxLayout" name="mainLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="logoWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item>
          <widget class="QLabel" name="labelLogo">
           <property name="pixmap">
            <pixmap resource="ImgSrc.qrc">:/img/mainLogo.png</pixmap>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="alcohol">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="icon">
            <iconset resource="ImgSrc.qrc">
             <normaloff>:/img/misc/Alcohol_120.png</normaloff>:/img/misc/Alcohol_120.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>36</width>
             <height>36</height>
            </size>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="setupButton">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>系统设置</string>
           </property>
           <property name="icon">
            <iconset resource="ImgSrc.qrc">
             <normaloff>:/img/misc/setup.png</normaloff>:/img/misc/setup.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>36</width>
             <height>36</height>
            </size>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="defaultHome" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ImgSrc.qrc"/>
 </resources>
 <connections/>
</ui>
